src/main/java/user/sort/ISortComparator.java

```diff
@@ -0,0 +1,11 @@
+package user.sort;
+import entity.Review;
+
+public interface ISortComparator {
+
+    int compareToHelpful(Review otherReview);
+
+    int compareToHighestRating(Review otherReview);
+
+    int compareToNewest(Review otherReview);
+}
```

Comment:
I don't know if I just missed it, but it doesn't look like any of your classes implement this interface. If that's the case, you can just delete it (this has nothing to do with functionality, just organization)

Reply:
Nevermind, you changed the Review class, I did not see that